### ISSUE TITLE
Site removal: moddrop.com

### ISSUE THREAD

**Issue by berrysauce** [VictiniX888/Illegal-Mod-Sites#518]
Reported to StopModReposts: "got a message that https://www.moddrop.com/stardew-valley/mods/487802-immersive-characters-shane is illegal redistribution but it’s posted by tenthousandcats, the original mod author"

Image shows mention of the site by the mod author on Discord:
![image](https://user-images.githubusercontent.com/38889179/169974563-e3036ff1-f359-4985-86f3-59f2b5ab6449.jpeg)

**Comment by Brandonbr1** [comment]
Yes I feel like there should be something lower than a straight up repost just saying that this website contains some reposts but has original 
content it is sort of like mcpedl.com where is it not really their fault it is just the users fault because they have no control to it as souppose to 9miecraft where the owners post the reposts with no community

**Comment by berrysauce** [comment]
Websites with content submitted by users still need to be moderated. A forum owner whose forum is used to share pirated movies can still be sued even though he didn't share any pirated movies, but the users did. That's why we're still listing such sites and call them reposting sites.

**Comment by GEOEGII555** [comment]
> Websites with content submitted by users still need to be moderated. A forum owner whose forum is used to share pirated movies can still be sued even though he didn't share any pirated movies, but the users did. That's why we're still listing such sites and call them reposting sites.

That means you can't go by mod's author officially posted links, and what's wrong with moddrop

**Comment by Brandonbr1** [comment]
> Reported to StopModReposts: "got a message that https://www.moddrop.com/stardew-valley/mods/487802-immersive-characters-shane is illegal redistribution but it’s posted by tenthousandcats, the original mod author"
> 
> Image shows mention of the site by the mod author on Discord: ![image](https://user-images.githubusercontent.com/38889179/169974563-e3036ff1-f359-4985-86f3-59f2b5ab6449.jpeg)

This site is in a gray area because it's Stardew valley and terria catagory has original mods

It's only the Minecraft section that has an issue with reposting
